app/services/triannon/ldp_loader.rb

```diff
@@ -99,7 +99,7 @@ def get_ttl sub_path = nil
     def statements_from_ttl_minus_fedora ttl
       # RDF::Turtle::Reader.new(ttl).statements.to_a
       g = RDF::Graph.new.from_ttl(ttl) if ttl
-      RDF::FCRepo4.remove_fedora_triples(g).statements if g
+      OA::Graph.remove_fedora_triples(g).statements if g
```

Comment:
I guess the concern here boils down to whether the method can return `nil` or always returns an RDF::Graph, even if the RDF::Graph might be empty.  I think it currently returns `nil` when `ttl` is `nil` or it throws an exception when trying to parse the `ttl` with RDF::Graph.new.from_ttl(ttl)`.